apps/bestofjs-nextjs/src/app/api/og/projects/route.tsx

```diff
@@ -0,0 +1,96 @@
+import { formatNumber } from "@/helpers/numbers";
+import { TagIcon } from "@/components/core";
+import {
+  ProjectPageSearchParams,
+  parseSearchParams,
+} from "@/components/project-list/navigation-state";
+import {
+  SortOption,
+  SortOptionKey,
+  sortOrderOptionsByKey,
+} from "@/components/project-list/sort-order-options";
+import { api } from "@/server/api-remote-json";
+import { Box, generateImageResponse, mutedColor } from "@/app/api/og/og-utils";
+
+import { ImageLayout } from "../og-image-layout";
+import { ProjectRow } from "../og-utils";
+
+export const runtime = "edge";
+
+export async function GET(_req: Request) {
```

Comment:
It's better to avoid the prefix `_` as it sometimes conveys a special meaning.

Example: https://nextjs.org/docs/app/building-your-application/configuring/draft-mode#step-1-create-and-access-the-route-handler